docs/internal_api.yml

```diff
@@ -512,6 +512,38 @@ paths:
         500:
           $ref: "#/responses/InternalServerError"
 
+  /device/deployments/last:
```

Comment:
the path is like nothing else in this service, but maybe it's OK; I don't better idea anyway...

Reply:
where we'll take the tenant id from?

Reply:
discussed privately and changed.